scripts/recording_uploader/deploy.py

```diff
@@ -0,0 +1,42 @@
+"""Entrypoint to deploy the uploader to AWS Lambda."""
+
+
+import pathlib
+import subprocess
+
+from loguru import logger
+import boto3
+import fire
+
+CURRENT_DIR = pathlib.Path(__file__).parent
+
+
+def main(region_name: str = "us-east-1", guided: bool = True) -> None:
+    """Deploy the uploader to AWS Lambda.
+
+    Args:
+        region_name (str): The AWS region to deploy the Lambda function to.
+        guided (bool): Whether to use the guided SAM deployment.
+    """
+    s3 = boto3.client(
```

Comment:
@KIRA009 can you please modify this to use the credentials specified in `openadapt.config`? Specifically we should add `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` or similar.

Also, please add some documentation regarding the permissions required for this IAM user.

Reply:
Do we want to add these keys to the config? They won't be used anywhere else in the project, and I think when you run the deploy script, if boto3 doesn't find appropriate keys in the place where its looking, the user is notified of that.

For the access keys, I followed this without any explicit permissions - https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey. These keys are the ones that you need to run the deploy script, which then creates appropriate iam users on its own. Once its deployed, if you want you can delete the original access keys. Should I add this in the readme?